### PR TITLE
Sort equations & figures

### PR DESCRIPTION
**Pull request by Lawrencium77**

### CODE REVIEW

File: Fast and Accurate GPU Quantization for Transformers.md [hunk 1]
```diff
@@ -93,11 +92,7 @@ To perform calibration, we used TensorRT’s [PyTorch Quantization Toolkit](http
 
 A final distinction to be made is how quantization parameters are shared between elements of our parameters and activations. Throughout this blog, we'll use the following diagram to illustrate a matmul:
```
Comment (david-macleod) [inline Fast and Accurate GPU Quantization for Transformers.md]: We don't really use it throughout the blog? Do we wanna re-word?

Comment (david-macleod) [reply Fast and Accurate GPU Quantization for Transformers.md]: Oh I think I get what is meant now, we use W, X, Y later on. We might want to remove this comment anyway if we are planning to move this section after Specifics of INT8 GEMMs (where the equations are defined).

Comment (david-macleod) [reply Fast and Accurate GPU Quantization for Transformers.md]: Can we change line 99 from `It’s also feasible to share quantization parameters` to `It’s also common to share quantization parameters`

File: Fast and Accurate GPU Quantization for Transformers.md [hunk 2]
```diff
@@ -77,13 +74,15 @@ Static quantization involves obtaining activation quantization parameters by pas
 There are multiple methods to derive a clipping range from these activations, such as:
 
 * Taking the min/max value from the calibration data
+* Taking some percentile (e.g 99.99%) to determine the max value
 * Minimising KL Divergence between the input and quantized distributions
 * Minimising the Mean-Squared Error between input and quantized distributions
 
-To perform calibration, we used TensorRT’s [PyTorch Quantization Toolkit](https://github.com/NVIDIA/TensorRT/tree/master/tools/pytorch-quantization). Another option is to use the `QuantStub` and `DeQuantStub` nodes from [PyTorch](https://pytorch.org/docs/stable/quantization.html) directly, to capture the relevant statistics.
+The following figure, taken from an Nvidia whitepaper [5], shows a histogram of input activations for some layer in a neural network. The vertical lines represent the maximum clipping range, $\beta$, for various calibration schemes:
```
Comment (david-macleod) [inline Fast and Accurate GPU Quantization for Transformers.md]:
```suggestion
The following figure [5] shows a histogram of input activations for some layer in a neural network. The vertical lines represent the maximum clipping range, $\beta$, for various calibration schemes:
```

File: Fast and Accurate GPU Quantization for Transformers.md (hunk 1 above)
Comment (Lawrencium77) [reply Fast and Accurate GPU Quantization for Transformers.md]: Have gone with both of these

**Pull request merged**